sample-code/src/server.ts

```diff
@@ -738,3 +739,19 @@ app.get('/prompt-registry/template', async (req, res) => {
     sendError(res, error);
   }
 });
+
+app.get('/test/deployment-id', async (req, res) => {
+  try {
+    // Example: test with a model name (foundation-models)
+    const deploymentId1 = await getFoundationModelDeploymentId(
```

Comment:
[req] This test is not necessary. We already have a test at `/ai-api/deployments` which tests `getDeployments()` method.
What you need to test as part of this ticket is:
1. Add a new type test that user can pass a deploymentId to the orchestrationClient initialization. 
2. Add a unit test to check that if you set a `deploymentId`, then the resolve deployment method is not called. This should be possible with jest's `spyOn`.